src/Platforms/OraclePlatform.php

```diff
@@ -313,9 +317,62 @@ public function getListSequencesSQL(string $database): string
      */
     protected function _getCreateTableSQL(string $name, array $columns, array $options = []): array
     {
-        $indexes            = $options['indexes'] ?? [];
-        $options['indexes'] = [];
-        $sql                = parent::_getCreateTableSQL($name, $columns, $options);
+        $columnListSql = $this->getColumnDeclarationListSQL($columns);
+
+        if (isset($options['uniqueConstraints']) && ! empty($options['uniqueConstraints'])) {
+            foreach ($options['uniqueConstraints'] as $definition) {
+                $columnListSql .= ', ' . $this->getUniqueConstraintDeclarationSQL($definition);
+            }
+        }
+
+        if (isset($options['primary']) && ! empty($options['primary'])) {
+            $columnListSql .= ', PRIMARY KEY(' . implode(', ', array_unique(array_values($options['primary']))) . ')';
+        }
+
+        $temporary = match ($options['temporary'] ?? '') {
+            '' => '',
+            'global' => 'GLOBAL TEMPORARY ',
+            'private' => 'PRIVATE TEMPORARY ',
+            default => throw new InvalidArgumentException(sprintf(
+                'invalid temporary specification for table %s',
+                $name,
+            ))
+        };
```

Comment:
Same question here. Why are my options for `temporary` different for each platform?